patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -1,34 +1,43 @@
 --- ../src-base/minecraft/net/minecraft/entity/player/EntityPlayer.java
 +++ ../src-work/minecraft/net/minecraft/entity/player/EntityPlayer.java
-@@ -96,6 +96,11 @@
+@@ -96,6 +96,12 @@
  @SuppressWarnings("incomplete-switch")
  public abstract class EntityPlayer extends EntityLivingBase
  {
 +    public static final String PERSISTED_NBT_TAG = "PlayerPersisted";
 +    protected java.util.HashMap<Integer, BlockPos> spawnChunkMap = new java.util.HashMap<Integer, BlockPos>();
 +    protected java.util.HashMap<Integer, Boolean> spawnForcedMap = new java.util.HashMap<Integer, Boolean>();
 +    public float eyeHeight = this.getDefaultEyeHeight();
++    public static final net.minecraft.entity.ai.attributes.IAttribute REACH_DISTANCE = new net.minecraft.entity.ai.attributes.RangedAttribute(null, "generic.reachDistance", 5.0D, 0.0D, 1024.0D).func_111112_a(true);
```

Comment:
The discussion about adding retroactively to vanilla is out of the scope of this PR (and probably out of scope for forge). Reducing the number of patches is a good enough reason for me, its public so it can still be used elsewhere anyway.